python_modules/libraries/dagster-databricks/dagster_databricks/components/databricks_asset_bundle/component.py

```diff
@@ -73,4 +74,17 @@ def get_asset_spec(self, task: DatabricksBaseTask) -> AssetSpec:
         )
 
     def build_defs(self, context: ComponentLoadContext) -> Definitions:
-        raise NotImplementedError()
+        component_defs_path_as_python_str = str(
+            os.path.relpath(context.defs_module_path, start=context.project_root)
```

Comment:
this will still be the same for all instances of this component in the project because defs_module_path is the path of the top-level defs_module. you can use `context.component_path` instead.

Reply:
Fixed in [553dc2f](https://github.com/dagster-io/dagster/pull/31712/commits/553dc2ffecbeca75e771a08d67b811fe908c6635)